### ISSUE TITLE
Nitpicks on your very well-made app

### ISSUE THREAD

**Issue by avocadoboi** [roldanjr/pomatez#157]
Hi! I have two very minor suggestions that might be a little subjective. 

In the timer screen, remove the "There's no description yet added on this card." text when there is no description on the current card in the priority list. To me it creates a small distraction, or the feeling that I need to add a description when I just want the title as a note there.

Secondly, I would either change the session counter to say "Session 1/4", or make it count from zero: "0/4 Sessions". I think most users get a little confused when it says "3/4 Sessions" and they have two more sessions to go, as the natural interpretation at least to me is "3/4 sessions completed".

Your app deserves more attention, it is so well designed, I love it! :^)

**Comment by saumyasinghal747** [comment]
I agree with both of these suggestions, they were both things that I noticed right as I started using the app. I think these improvements would really add to the professionalism of Pomatez.

Another thing I think should be done is to display the name of the priority list somewhere in the timer screen or in the menu bar so that when I see my task I know what the context is. Sometimes I come back after a long break and I see the task "Make Button" but I do not know what I need to make a button for. I use a list for every "feature" and these features might have the tasks "Make Button", "Make function", and "Connect button to function". Retyping the name of the feature before every task name is cumbersome and I'd appreciate if it can be shown, even if it is super small, somewhere in the Timer page.
